### ISSUE TITLE
Allow multiple modifier keys [feature request]

### ISSUE THREAD

**Issue by tierschicksale** [felixhageloh/uebersicht#135]
Would it be possible to allow uebersicht to respond with different afterRender actions when different modifier keys are used with the same item?

**Comment by felixhageloh** [comment]
I think I don't fully understand. By modifier keys, do you mean keys like `⌘` or `⌥` that you press while clicking something? 
I'm asking because `afterRender` doesn't get called after clicking something

**Comment by tierschicksale** [comment]
Yes, those are the modifier keys I'm talking about. I would like one command to run if I'm pressing the ⌘ key while I click on an object, while a different one runs if I'm pressing the ⌥ key and I click on that same object.  Uebersicht would have to have some way of distinguishing which modifier key was being held down while the user clicked on the object.

- tierschicksale
